tests/message_ut/message_ut.c

```diff
@@ -165,49 +169,46 @@ TEST_FUNCTION(when_allocating_memory_for_the_message_fails_then_message_create_f
 TEST_FUNCTION(message_clone_with_a_valid_argument_succeeds)
```

Comment:
There should be one test to check for general failures, using umock_c_negative_tests_*